processor/src/errors.rs

```diff
@@ -61,6 +62,9 @@ impl Display for ExecutionError {
                     "Failed to execute code block with root {hex}; the block could not be found"
                 )
             }
+            CycleLimitExceeded(max_cycles) => {
+                write!(f, "Exceeded the allowed number of cycles which equals {max_cycles}")
```

Comment:
minor nit: I'd change the text to "Exceeded the allowed number of cycles (max cycles = {max_cycles})"